ui/pages/confirmations/components/confirm/info/batch/transaction-account-details/transaction-account-details.tsx

```diff
@@ -5,49 +5,50 @@ import {
   ConfirmInfoRowAddress,
   ConfirmInfoRowText,
 } from '../../../../../../../components/app/confirm/info/row';
-import { useConfirmContext } from '../../../../../context/confirm';
 import { ConfirmInfoSection } from '../../../../../../../components/app/confirm/info/row/section';
-import {
-  useIsDowngradeTransaction,
-  useIsUpgradeTransaction,
-} from '../../hooks/useIsUpgradeTransaction';
 import { useI18nContext } from '../../../../../../../hooks/useI18nContext';
+import { useConfirmContext } from '../../../../../context/confirm';
+import { useIsUpgradeTransaction } from '../../hooks/useIsUpgradeTransaction';
+import { SmartContractWithLogo } from '../../shared/smart-contract-with-logo';
 
 export function TransactionAccountDetails() {
   const t = useI18nContext();
   const { currentConfirmation } = useConfirmContext<TransactionMeta>();
   const isUpgrade = useIsUpgradeTransaction();
-  const isDowngrade = useIsDowngradeTransaction();
   const { chainId, txParams } = currentConfirmation;
   const { from } = txParams;
 
-  if (!isUpgrade && !isDowngrade) {
```

Comment:
Is this not useful to guard against misuse of this component as we shouldn't render that the account type is changing if it's not?

Reply:
Fixed